src/js/popup.js

```diff
@@ -1,17 +1,20 @@
-(function($, chromeService){
+(function($, chromeService, nowValueService){
 
   var _public = {};
   var alertList, alertListContainer;
-  
+
   function init(){
     bindElements();
     getSavedAlerts();
+    getNowValue();
   }
 
   function bindElements(){
     $('[data-js="button-save"]').on('click', onButtonSaveClick);
     alertListContainer = $('[data-js="alert-list-container"');
     alertList = $('[data-js="alert-list"');
+    nowValueField = $('[data-js="now-value-field"');
+    nowValueVariation = $('[data-js="now-value-variation"');
```

Comment:
These two variables are going to the global namespace. Declare them in the line 4 like it has been done for `alertList` and `alertListContainer `.